src/main/java/org/joshsim/engine/geometry/Grid.java

```diff
@@ -0,0 +1,45 @@
+package org.joshsim.engine.geometry;
+
+import java.util.List;
+import org.joshsim.engine.entity.Patch;
+import org.joshsim.engine.value.EngineValue;
+
+/**
+ * The Grid class contains a list of patches and a cell width for conveniences.
+ * Spatial operations will be performed on the elements of the grid, Patches, which
+ * are the basic unit of spatial representation.
+ */
+public class Grid {
+  private final List<Patch> patches;
+  private final EngineValue spacing;
+
+
+  /**
+   * Constructor for the Grid class.
+   *
+   * @param patches List of patches in the grid.
+   * @param spacing Width of a cell in the grid.
+   */
+  public Grid(List<Patch> patches, EngineValue spacing) {
+    this.patches = patches;
```

Comment:
For a later merge request, we might want to index into latitude and longitude with TreeMap.